twopaneview/src/types.ts

```diff
@@ -15,5 +15,5 @@ export enum PanePriority {
 export enum PaneMode {
     Auto = 'Auto',
     Single = 'Single',
-    Dual = 'Dual',
+    Double = 'Double',
```

Comment:
Might want to double-check (no pun intended) with Kiki and Yulia about the naming here.
I do agree "Double" makes more emphasis on the behavior of two pieces of content on a single screen.

Reply:
in surface duo they use Dual

Reply:
The spec has 'Double' so I'd prefer to stay with the spec.  I'll circle back with PM with this as an open question, along with the DeviceOrientation proposal.

---
In reply to: [426024994](https://github.com/microsoft/react-native-dualscreen/pull/34#discussion_r426024994) [](ancestors = 426024994)